Frameworks/UIKit/UIPasteboard.mm

```diff
@@ -34,12 +34,10 @@
 NSString* const UIPasteboardChangedTypesAddedKey = @"UIPasteboardChangedTypesAddedKey";
 NSString* const UIPasteboardChangedTypesRemovedKey = @"UIPasteboardChangedTypesRemovedKey";
 
-/** @Status Stub */
-NSArray* const UIPasteboardTypeListString = StubConstant();
-/** @Status Stub */
-NSArray* const UIPasteboardTypeListURL = StubConstant();
-/** @Status Stub */
-NSArray* const UIPasteboardTypeListImage = StubConstant();
+NSArray<NSString*>* const UIPasteboardTypeListString = @[ (NSString*)kUTTypeUTF8PlainText, (NSString*)kUTTypeText ];
+NSArray<NSString*>* const UIPasteboardTypeListURL = @[ (NSString*)kUTTypeURL ];
+NSArray<NSString*>* const UIPasteboardTypeListImage = @[ (NSString*)kUTTypePNG, (NSString*)kUTTypeJPEG ];
+
```

Comment:
These three arrays are autoreleased and will be automatically destroyed when the pool containing them goes out of scope.

Please set them with `retain`ed values